hsmd/libhsmd.c

```diff
@@ -350,7 +351,7 @@ static u8 *handle_ready_channel(struct hsmd_client *c, const u8 *msg_in)
 	struct amount_msat value_msat;
 	struct channel_type *channel_type;
 
-	if (!fromwire_hsmd_ready_channel(tmpctx, msg_in, &is_outbound,
+	if (!fromwire_hsmd_setup_channel(tmpctx, msg_in, &is_outbound,
```

Comment:
The name of the enclosing routine (on line 337) should also be changed to `handle_setup_channel`

(I'm not sure how to attach review comments to lines that didn't change)

Reply:
>(I'm not sure how to attach review comments to lines that didn't change)

It is a little bit hard with github, much nicer the gitlab one


Thanks, you are right I forget all these small code things! thanks for the good catch